jenkins/spark-nightly-build.sh

```diff
@@ -27,12 +27,16 @@ MVN="mvn -Dmaven.wagon.http.retryHandler.count=3 -DretryFailedDeploymentCount=3
 
 DIST_PL="dist"
 function mvnEval {
-    $MVN help:evaluate -q -pl $DIST_PL $MVN_URM_MIRROR -Prelease311 -Dmaven.repo.local=$M2DIR -Dcuda.version=$CUDA_CLASSIFIER -DforceStdout -Dexpression=$1
+    $MVN help:evaluate -q -pl $DIST_PL $MVN_URM_MIRROR -Prelease311 -Dmaven.repo.local=$M2DIR -DforceStdout -Dexpression=$1
 }
 
 ART_ID=$(mvnEval project.artifactId)
 ART_GROUP_ID=$(mvnEval project.groupId)
 ART_VER=$(mvnEval project.version)
+CUDA_CLASSIFIER=${CUDA_CLASSIFIER:-$(mvnEval cuda.version)} # default cuda version
```

Comment:
can we call this DEFAULT_CUDA_CLASSIFIER?

Reply:
yes, just need to update pipeline ENV. I will rename it